autoshop/analytics/processor.py

```diff
@@ -0,0 +1,4 @@
+def related_website(request):
+
+    related_site = {'PakWheels_site': 'https://www.pakwheels.com/'}
```

Comment:
this is not what we wanted. Just pass django's settings and site object directly